src/core/internal/execinfo.d

```diff
@@ -0,0 +1,312 @@
+/**
+ * This module helps to decide whether an appropriate execinfo implementation
+ * is available in the underling C runtime or in an external library. In the
+ * latter case exactly one of the following version identifiers should be
+ * set at the time of building druntime.
+ *
+ * Possible external execinfo version IDs based on possible backtrace output
```

Comment:
This should be in a release note, it's relevant to packagers (e.g. for Musl)

Reply:
Done.